src/ui/components/Toolbar.tsx

```diff
@@ -156,6 +165,15 @@ export default function Toolbar() {
   return (
     <div className="toolbox-toolbar-container flex flex-col items-center justify-between py-1">
       <div id="toolbox-toolbar">
+        {recording?.metadata?.test?.runner?.name !== "cypress" && showTour ? (
```

Comment:
`useTestInfo` will give you the name as `runner` as well if you want to simplify this lookup.

Reply:
Will address in follow-up, thanks!